Add javadoc for ServerPing.Builder

Andre601 opened the pull request:
Adds Javadoc comments to `ServerPing.Builder` methods to explain them a little bit.

I tried to keep it brief and only focused on the setter methods.

Andre601 committed:
Add javadoc for ServerPing.Builder

Andre601 commented:
That checkstyle action seems broken...
Because it reports issues that aren't there. For example:

- `ServerPing.java:162` points to `/**`
- `ServerPing.java:165` is reported as a `@param` missing a preceding empty line... which it doesn't.

I really don't know why it gives such blatant false positives here.

electronicboy commented:
maybe the whitespace after the `*` on that line? 162 is also the start of that JD comment, of which the first line is infact missing a period; Also, `this builder`*

Unable to go over it fully, but, some of the descriptions feel a bit, flat

Andre601 committed:
Address codestyle reports

Andre601 commented:
> maybe the whitespace after the `*` on that line? 162 is also the start of that JD comment, of which the first line is infact missing a period; Also, `this builder`*

Ngl, kinda dumb if it actually complains about a single space after a `*` but it's codestyle, so whatever... Addressed that.
Also the `this build` part was a copy-paste of the `mods(ModInfo)` javadoc that already existed, so that was there before. Fixed it.

> Unable to go over it fully, but, some of the descriptions feel a bit, flat

Feel free to suggest better lines here. Like I said did I keep it to a minimum and honestly are some methods too foregin to me to properly describe (i.e. the mod-related methods).
